inc/azure_c_shared_utility/tlsio_appleios.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef TLSIO_APPLEIOS_H
+#define TLSIO_APPLEIOS_H
+
+// DEPRECATED: tls_config will be removed from the tree.
```

Comment:
This does not need to be here as a comment. ALso, why remove TLSIO_CONFIG? It is the structure with the config which is common between TLS_IO implementations. Namingly host, port, optional underlying IO.

Reply:
I built this adapter by modifying the Arduino adapter. These are artifacts from the original code. Not a defense, just an explanation.

---
In reply to: [139021976](https://github.com/Azure/azure-c-shared-utility/pull/109#discussion_r139021976) [](ancestors = 139021976)

Reply:
Removed this header. It only has one define that is not required.

---
In reply to: [139216474](https://github.com/Azure/azure-c-shared-utility/pull/109#discussion_r139216474) [](ancestors = 139216474,139021976)